docs/content/Schema/Reference/pre-aggregations.mdx

```diff
@@ -442,7 +442,7 @@ scenarios where real-time data is required.
 
 [Lambda pre-aggregations][ref-caching-lambda-preaggs] can be used to combine
 data from a data source and a pre-aggregation, or even from multiple
-pre-aggregations across different schema-compatible cubes.
+pre-aggregations across different data model-compatible cubes.
```

Comment:
Just saying: I have no idea what "data model-compatible cubes" are.

Reply:
Changed to:

> or even from multiple pre-aggregations across different cubes that share the same dimensions
and measures.